app/models/stored_result.rb

```diff
@@ -0,0 +1,11 @@
+class StoredResult < ApplicationRecord
+  def self.write(result_key, result_value)
```

Comment:
should it also accept `expire_at` info?

Reply:
Yeah, it should for sure, thanks. It is defaulting to 24 hours.